book/thematics/events/graphs/futureEvent.json

```diff
@@ -0,0 +1,78 @@
+{
+    "@context": {
+        "@vocab": "https://schema.org/"
+    },
+    "@type": "Event",
+    "@id": "https://pidOfThisJsonLDdoc.org/",
+    "name": "The event name",
+    "EventStatus": "EventScheduled",

```

Comment:
This property is what we would use (together with the start/endDate to express this is a future or planned event. Some concerns being clarified in https://github.com/schemaorg/schemaorg/issues/3236